src/services/homework/hooks/submissions.js

```diff
@@ -238,33 +213,44 @@ const preventNoTeamMember = (context) => {
 		context.data.teamMembers = [context.params.account.userId];
 	}
 	if (context.method == 'update' && (!context.data.teamMembers || context.data.teamMembers.length == 0)) {
-		return Promise.reject(new Conflict({
-			message: 'Abgaben ohne TeamMember sind nicht erlaubt!',
-		}));
+		return Promise.reject(
+			new Conflict({
+				message: 'Abgaben ohne TeamMember sind nicht erlaubt!',
+			})
+		);
 	}
 	return Promise.resolve(context);
 };
 
 const setTeamMembers = (context) => {
-	if (!context.data.teamMembers) { // if student (no grading) is going to submit without teamMembers set
+	if (!context.data.teamMembers) {
+		// if student (no grading) is going to submit without teamMembers set
 		context.data.teamMembers = [context.params.account.userId];
 	}
 	if (!context.data.teamMembers.includes(context.params.account.userId)) {
-		return Promise.reject(new Conflict({
-			message: 'Du kannst nicht ausschließlich für andere Abgeben. Füge dich selbst zur Abgabe hinzu!',
-		}));
+		return Promise.reject(
+			new Conflict({
+				message: 'Du kannst nicht ausschließlich für andere Abgeben. Füge dich selbst zur Abgabe hinzu!',
+			})
+		);
 	}
 };
 
 const noSubmissionBefore = (context) => {
-// check that no one has already submitted for the current User
-	const submissionsForMe = context.data.submissions.filter((submission) => // is there an submission for the current user?
-		(submission.teamMembers.includes(context.params.account.userId))
-		|| ((submission.studentId || {})._id == context.params.account.userId));
+	// check that no one has already submitted for the current User
+	const submissionsForMe = context.data.submissions.filter(
+		(
+			submission // is there an submission for the current user?
+		) =>
+			submission.teamMembers.includes(context.params.account.userId) ||
+			(submission.studentId || {})._id == context.params.account.userId
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)